### ISSUE TITLE
Feedback for “Quickstart - macOS”

### ISSUE THREAD

**Issue by erikkristoferanderson** [cloudquery/cloudquery#15510]
Under section "Start Syncing" there is an incorrect or missing word.

It occurs in this text:

"You can sync the by specifying a directory with the configuration files or specify files directly"

It's not obvious to me what the correct text should say, so I'm opening an issue for discussion.

**Comment by erezrokah** [comment]
Good catch @erikkristoferanderson, let me know if https://github.com/cloudquery/cloudquery/pull/15512 helps
